app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person_dict in people:
+        person = Person(person_dict["name"], person_dict["age"])
+        person_list.append(person)
+    for person_dict in people:
+        person = Person.people[person_dict["name"]]
+        if "wife" in person_dict and person_dict["wife"] is not None:
```

Comment:
You can simplify the check by using the .get() dictionary method. If the dictionary does not have such a key, it will return None. Don't forget to simplify this check elsewhere as well.